app/models/mod_warning.rb

```diff
@@ -2,13 +2,20 @@ class ModWarning < ApplicationRecord
   # Warning class name not accepted by Rails, hence this needed
   self.table_name = 'warnings'
 
-  belongs_to :community_user
+  scope :global, -> { where(is_suspension: true, is_global: true) }
```

Comment:
This scope does not appear to be used anywhere (including in other files). Is this being used in a way I can't find? If not, should this be left here for potential future use or be removed as redundant?